README.md

```diff
@@ -112,6 +113,78 @@ $ remme account get-balance \
 368440.0
 ```
 
+### Transaction
+
+Get list of transaction — ``remme transaction get-list``:
+
+| Arguments   | Type   |  Required | Description                                            |
+| :---------: | :----: | :-------: | -----------------------------------------------------  |
+| ids         | String |  No       | Identifiers to get a list of transaction by.          |
+| start       | String |  No       | Id to start paging (inclusive).                        |
+| limit       | Integer|  No       | Number of transaction to return.                       |
+| head        | String |  No       | Id of head block.                                      |
+| reverse     | String |  No       | Reverse result.                                        |
+| node-url    | String |  No       | Node URL to apply a command to.                        |
+| family-name | String |  No       | List transaction by its family name.                   |
```

Comment:
Only one, because of the family name in core its [String field](https://github.com/Remmeauth/remme-core/blob/dev/remme/shared/forms/transaction.py#L24).